program/src/main/System.vala

```diff
@@ -125,12 +136,48 @@ public class System
 
             
             //right side: marks + cgi + hotkeys
+            var util_pane = new Gtk.Paned(Gtk.Orientation.VERTICAL);
+
+            //put marksGrid in scrollable in case of overflow
+            var scroll = new Gtk.ScrolledWindow(null,null);
+            scroll.set_size_request(230,500);
+            scroll.set_propagate_natural_width(true);
+
             marksGrid = new Gtk.Grid();
             marksGrid.set_row_spacing(5);
             marksGrid.set_column_spacing(5);
-            marksGrid.set_size_request(100, -1); //100 = minimum width
-            hpaned.pack2(marksGrid, false, false); //put the vpaned on the right side of the hpaned
+            marksGrid.set_size_request(230, 500); //100 = minimum width
```

Comment:
Was the minimum width here updated?

Reply:
Yes, iirc it was updated to account for the addition of the side taskbar just to let the UI elements breath a little bit more. Let me know if this breaks anything that I didn't catch. I will update the comment accordingly